### DISCUSSION TITLE
Calculate Ditance

### DISCUSSION THREAD

**Discussion by zoolle** [rnmapbox/maps#1384] [Q&A] [answered]
I need to calculate the distance between two points and by using this.

`const time_to_pickup = MapboxGL.geoUtils.calculateDistance(
      [currentLocation.longitude, currentLocation.latitude],
      [order.origin.longitude, order.origin.latitude],
      'meters',
);`

I get this 

`TypeError: undefined is not an object (evaluating '_maps.default.geoUtils.calculateDistance')`

Is there a different setup?

**Comment by ferdicus** [comment, marked as answer]
Please use turf instead: https://turfjs.org/docs/#distance
Our geoutils are subject to change and internally they use turf anyways.
